src/fragments/start/getting-started/ios/setup.mdx

```diff
@@ -53,25 +51,21 @@ Amplify for iOS is distributed through Swift Package Manager, which is integrate
   amplify init --quickstart --frontend ios
   ```
 
-1. Switch back to Xcode. Select **File > Swift Packages > Add Package Dependency**.
+1. Switch back to Xcode. Select **File > Add Packages...**
 
     ![Add package dependency](/images/project-setup/20_4_add-package-dependency.png)
 
-1. Enter the Amplify iOS GitHub repo URL (`https://github.com/aws-amplify/amplify-ios`) into the search bar and click Next.
+1. Enter the Amplify iOS GitHub repo URL (`https://github.com/aws-amplify/amplify-ios`) into the search bar and wait for the result to load. You'll see the Amplify iOS repository rules for which version of Amplify you want Swift Package Manager to install. Choose the dependency rule as , **Up to Next Major Version**, as it will use the latest compatible version of the dependency that can be detected from the `main` branch, then click **Add Package**.
```

Comment:
```suggestion
1. Enter the Amplify iOS GitHub repo URL (`https://github.com/aws-amplify/amplify-ios`) into the search bar and hit Enter. Wait for the result to load. You'll see the Amplify iOS repository rules for which version of Amplify you want Swift Package Manager to install. Choose the dependency rule as , **Up to Next Major Version**, as it will use the latest compatible version of the dependency that can be detected from the `main` branch, then click **Add Package**.
```